app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt

```diff
@@ -155,7 +154,7 @@ class MenuDrawerFragment : MenuFoldersFragment(), MailboxListFragment {
 
         syncAutoConfig.setOnClickListener {
             trackSyncAutoConfigEvent("openFromMenuDrawer")
-            startActivity(Intent(context, SyncAutoConfigActivity::class.java))
+            requireActivity().launchSyncAutoConfigActivityForResult()
```

Comment:
it seems you only use this function in fragments, why not putting the requireActivity() diurectly in the extension, changing it to Fragment extension